### ISSUE TITLE
Forgot Password option isn't working

### ISSUE THREAD

**Issue by GruLab** [kimai/kimai#941]
#### Expected behaviour
Password Reset Link should be sent to the registered mail-id.

#### Actual behaviour
Nothing is being sent to the registered mail-id.

#### Steps to reproduce
1. Open Kimai Home Page
2. Click on "Forgot my password" link
3. Enter registered Username >> Click "Reset Password" button

#### Reproducable on demo.kimai.org? [yes|no]
no,as i amn't a registered user on demo.kimai.org

#### Kimai version [for example: dev 1.1.0.1389|github master|github develop]
Kimai 1.1.0.1389 - Stable

#### Relevant log entries from log file (debug tab or temporary/logfile.txt)
[30.03.2017 15:36:23] password reset: kaniskan as user

**Comment by timlegge** [comment]
What authentication method are you using?  I appears that it is only supported with the built in kimai authentication support

**Comment by timlegge** [comment]
I just tried it out and I was missing 2 things.  

1) Did not have the admin email on the advanced tab configured
2) I did not have sendmail installed

Once it was installed kimai delivered the message to sendmail.  Sendmail is not properly configured so it is not sent.

YOu can check the apache logs for issues in kimai (like the admin email address) and send mail not installed

You can check the sendmail logs to see if the message got to the sendmail daemon.

So in short kimai authentication appears to be able to send an email message to a sendmail daemon which will likely send it if its properly configured.

**Comment by GruLab** [comment]
@timlegge Thanks a ton for detailed info, i'll install sendmail & let you know regarding any further issues.

**Comment by timlegge** [comment]
No problem.  The patch at https://github.com/kimai/kimai/issues/944 supports sending via a smtp server instead.  Looks like I need to fix a couple a things around the installer to collect details and currently saving the config overwrites the settings but if you are not in a rush I will likely have smtp working shortly
